src/printer_tree.c

```diff
@@ -2488,6 +2488,29 @@ tro_read_module_name(const struct trt_tree_ctx *tc)
     return ret;
 }
 
+/**
+ * @brief Create implicit "case" node as parent of @p node.
+ * @param[in] node child of implicit case node.
+ * @return The case node ready to print.
+ */
+static struct trt_node
+tro_create_implicit_case_node(struct trt_node node)
+{
+    struct trt_node ret;
+
+    ret.status = node.status;
+    ret.flags = TRD_FLAGS_TYPE_EMPTY;
+    ret.name.type = TRD_NODE_CASE;
+    /* TODO: ret.name.module_prefix is not supported right now. */
```

Comment:
Why cannot you just use the prefix/name of the descendant?

Reply:
It is related to this -> [rfc](https://datatracker.ietf.org/doc/html/rfc8340):
>          If the node is augmented into the tree from another module,
>          its name is printed as <prefix>:<name>, where <prefix> is the
>          prefix defined in the module where the node is defined.

I haven't dealt with this prefix thing yet, because Radek once told me I didn't have to deal with it. I don't know if the situation has changed.
It's not the only place I don't deal with the prefix. This can be found variously in the printer tree code.

Reply:
However, I will add the prefix copy and delete the TODO. I will add the prefix implementation in another future PR.